src/js/ui/pane.js

```diff
@@ -1205,6 +1205,27 @@ $(function () {
                 } catch (e) {
                     CrmPowerPane.UI.ShowNotification("An error ocurred while redirecting to form editor.", "error");
                 }
+            });
+			
+			$("#open-webapi").click(function () {
+                try {
+					
+					var apiVersion = Xrm.Utility.getGlobalContext().getVersion();
+					var shortVersion= apiVersion.substring(3, apiVersion.indexOf(".") -1);
+					
+					Xrm.Utility.getEntityMetadata(Xrm.Page.data.entity.getEntityName(),"")
+						.then(function(result){
+							var url = Xrm.Page.context.getClientUrl() + "/api/data/v" + shortVersion + "/" + result.EntitySetName +"(" + Xrm.Page.data.entity.getId() + ")";
+							url = url.replace("{", "").replace("}","");
+					
+							window.open(url,'_blank');
+							
+						});
+                    
+                } catch (e) {
+                    CrmPowerPane.UI.ShowNotification("An error occured opening the Web API URL for this record.");
+                }
```

Comment:
Not sure if it's an editor/browser issue, but the indentation seems a bit off on the browser.